stdlib/TOML/src/TOML.jl

```diff
@@ -38,15 +38,10 @@ performance if a larger number of small files are parsed.
 """
 const Parser = Internals.Parser
 
-"""
-    DTParser()
-
-Constructor for a TOML `Parser` which returns date and time objects from Dates.
-"""
-function DTParser(args...; kwargs...)
-    parser = Parser{Dates}(args...; kwargs...)
-    return parser
-end
+# Dates-enabled constructors
+Parser() = Parser{Dates}()
```

Comment:
Type piracy I guess. Maybe that's ok.

Reply:
I left a comment behind in `base/toml_parser.jl` specifically delegating this definition to the TOML stdlib, so I think this is fine.